dcrtimed/backend/postgres/schema.md

```diff
@@ -0,0 +1,31 @@
+# Dcrtime Relational Database
+
+This document describes the SQL relational tables used in dcrtime.
+
+We have two tables storing all timestamped digests information, `records` 
+and `anchors` where the first is key/value like table used to store all 
+timestamped digests and the second for storing anchors information, where each 
+succesful anchor will result in a new entry. Each `record` which was included 
+in an `anchor` will be connected to the corresponding entry in the anchors 
+table using the col `anchor_merkle` which defined as forgein key & indexed in 
+records table, below you find the detailed description of the two tables:
+
+### Tables
+
+**Records:**
+| Col Name             | Type              | Not Null | P. Key | F. Key | Indexed | Unique | Description                  |
+|----------------------|-------------------|----------|--------|--------|---------|--------|------------------------------|
+| collection_timestamp | bigint            | x        |        |        | x       |        | Unix timestamp of collection |
+| digest               | bytea             | x        | x      |        | x       | x      | Timestamped digest           |
```

Comment:
What's the thought process for storing the anchor merkle as hex, but the digest as a byte array?  Why not store them as the same thing since they're both sha256 digests?

Reply:
Actually the `anchor_merkle` col is also `bytea`, but this file is outdated, my bad.
Same for the `merkle` col in the anchors table I changed it also to bytea.

Regarding `tx_hash` being a `text` is a mistake they all should be from the same type.
Then I guess they all should be `bytea` to make it easier to CURD them, sounds good?